packages/Collapsible/package.json

```diff
@@ -7,7 +7,7 @@
   "main": "lib/index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/acl-services/paprika/tree/master/packages/collapsible"
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/Collapsible"
```

Comment:
I think we should use homepage for this value and url property should point to the repo itself. https://github.com/acl-services/paprika/issues/91